### ISSUE TITLE
r5_dockerize:slurm doesn't work in containers 

### ISSUE THREAD

**Issue by themisa** [clustervision/trinity#444]
**Version:** r5_dockerize 38875bd
**Environment:** amaster

```
[root@c002 /]# sinfo    
sinfo: error: Unable to resolve "login": Host name lookup failure
sinfo: error: Unable to establish control machine address
slurm_load_partitions: Resource temporarily unavailable
```

**Comment by themisa** [comment]
Containers still think they are in vc-a instead of vc-new. It's possible some fixes were not merged yet in this branch
[root@c002 /]# hostname
c002.vc-a

**Comment by themisa** [comment]
This is also the case in vc-a

**Version:** r5_dockerize af84cae
**Environment:** dmaster

root@c001 /]# sinfo
sinfo: error: Unable to resolve "login": Host name lookup failure
sinfo: error: Unable to establish control machine address
slurm_load_partitions: Resource temporarily unavailable

**Comment by themisa** [comment]
This is a route on controller issue.

Fixed in 3db6123957

**Comment by abhimukh** [comment]
@themisa This should be fixed now.

**Comment by themisa** [comment]
Fixed in **Version:** r6 ea67ae8
